rpc/documentation/api.md

```diff
@@ -304,6 +306,8 @@ for querying for known used addresses).  Account discovery requires the wallet
 to be unlocked in order to derive account hardened extended pubkeys, and thus
 the private passphrase must be passed as a parameter when performing this
 action.  Account discovery is typically only required when reseeding a wallet.
+The address discover will begin at the provided starting block hash.  If none
+is provided, then the current network's GenesisHash will be used.
```

Comment:
"genesis hash". Use plain english in documentation instead of identifiers from the code unless specifically referring specifically to that identifier.